modules/tabix/bgzip/main.nf

```diff
@@ -23,8 +24,10 @@ process TABIX_BGZIP {
     in_bgzip = input.toString().endsWith(".gz")
     command1 = in_bgzip ? '-d' : '-c'
     command2 = in_bgzip ? ''   : " > ${prefix}.${input.getExtension()}.gz"
+    gzi_args = args.matches("(^| )-i\\b") ? "-I index/${prefix}.${input.getExtension()}.gz.gzi" : ''
     """
-    bgzip $command1 $args -@${task.cpus} $input $command2
+    mkdir -p index
```

Comment:
why does it need its own subfolder?

Reply:
Let me try again, but I think I tried first keeping everything in the same folder, and somehow the index was picked up by the "output" channel, not the "gzi" channel

Reply:
ah yes I can see why , the ${prefix}* will pick up everything. there is probably a regex way around it.

Reply:
It works ! Thanks for the poke